include/pybind11/pybind11.h

```diff
@@ -1389,6 +1405,63 @@ class module_ : public object {
         //       For Python 2, reinterpret_borrow was correct.
         return reinterpret_borrow<module_>(m);
     }
+
+    /** \rst
+        Initialized a module def for use with multi-phase module initialization.
+
+        ``def`` should point to a statically allocated module_def.
+        ``slots`` must point to an initialized array of slots with space for at
+           least one additional slot to be populated based on the options.
+    \endrst */
+    template <typename... Options>
+    static object init_module_def(const char *name,
+                                  const char *doc,
+                                  module_def *def,
+                                  PyModuleDef_Slot *slots,
+                                  Options &&...options) {
+        int i = 0;
```

Comment:
Could you please make this variable name slightly longer? E.g. `ix` would already be a big improvement, or `next_ix` maybe? Or alternatively, `PyModuleDef_Slot *next_slot = slots; while (next_slot->slot != 0) { next_slot++; }`?

Reply:
Done